_data/guides.yml

```diff
@@ -306,6 +306,6 @@ toc:
 - title: Administering Federation
   section:
   - title: Using `kubefed`
```

Comment:
Question for @kubernetes/sig-docs: ` ` doesn't render as code block in titles, should we keep this?